<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Myanmar Text&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>PyMOL HotSpots Plugin</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="step3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>STEP 3:  Color Spectrum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>BROWSE</string>
   </property>
   <property name="iconSize">
    <size>
     <width>33</width>
     <height>33</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="step2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>STEP 2:  Save All HotSpots</string>
   </property>
  </widget>
  <widget class="QLabel" name="step6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>STEP 6:     Search groups</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="box_step4_1">
    <item>
     <widget class="QLabel" name="label_radius1">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 7pt &quot;MS Shell Dlg 2&quot;;
</string>
      </property>
      <property name="text">
       <string>RADIUS     </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="radius1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(208, 208, 208);
selection-color: rgb(90, 181, 249);</string>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="box_step6">
    <item>
     <widget class="QLabel" name="label_radius2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 7pt &quot;MS Shell Dlg 2&quot;;
</string>
      </property>
      <property name="text">
       <string>RADIUS     </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="radius2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(208, 208, 208);
selection-color: rgb(90, 181, 249);</string>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="value">
       <double>3.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ok2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>620</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_hs">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
   <property name="iconSize">
    <size>
     <width>33</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="step1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>219</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>STEP 1:  Select Files</string>
   </property>
  </widget>
  <widget class="QLabel" name="step4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>219</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>STEP 4:    HotSpots Simplification</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>100</y>
     <width>20</width>
     <height>461</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="step5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>STEP 5:     Choose HotSpots</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="browser_files1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);
selection-color: rgb(118, 197, 230);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOAD</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="browser_files1_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);
selection-color: rgb(118, 197, 230);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>        HOTSPOTS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>        PROTEINS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>  +</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_files1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
   <property name="iconSize">
    <size>
     <width>33</width>
     <height>33</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="color_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>COLOR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_all">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>620</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="hs_browse">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);
selection-color: rgb(118, 197, 230);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_hs">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
   <property name="iconSize">
    <size>
     <width>33</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="save_file1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE FILE</string>
   </property>
  </widget>
  <widget class="QListWidget" name="colors">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(208, 208, 208);
color: rgb(255, 255, 255);
background-color: rgb(44, 44, 44);
selection-color: rgb(118, 197, 230);</string>
   </property>
   <item>
    <property name="text">
     <string>wheat_brown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lightorange_deepolive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sand_chocolate</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue_red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>white_black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rainbow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>slate_density</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>dirtyviolet_violetpurple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow_orange_red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>smudge_forest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>darksalmon_raspberry</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>palecyan_deepteal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lightpink_lightmagenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lightorange_orange</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lightblue_purpleblue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>limon_splitpea</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="save_file2_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE FILE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_file3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 101, 195);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SAVE FILE</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupSolvent">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(40, 40, 40);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="oneSolvent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>ONE SOLVENT</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="multipleSolvent">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>MULTIPLE SOLVENT</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupSimlification">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="geometric">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>GEOMETRIC CENTRE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="weights">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>HS WEIGHTS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="inz.qrc"/>
 </resources>
 <connections/>
</ui>
